ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -160,3 +160,26 @@ Get Data Connection Form Values
     ${s3_region}   get element attribute    ${S3_REGION_DC_INPUT_XP}      value
     ${s3_bucket}   get element attribute    ${S3_BUCKET_DC_INPUT_XP}      value
     [return]    ${s3_name}    ${s3_key}    ${s3_secret}    ${s3_endpoint}    ${s3_region}    ${s3_bucket}
+
+Recreate S3 Data Connection
```

Comment:
maybe the name is not very intuitive. It sounds like "delete the data connection and create it again", but it's not what id does